atomic_reactor/tasks/binary_container_build.py

```diff
@@ -119,23 +130,29 @@ def execute(self) -> None:
             image_size = podman_remote.get_image_size(dest_tag)
 
             if image_size > image_size_limit:
-                raise ExceedsImageSizeError(
+                exc = ExceedsImageSizeError(
                     'The size {} of image {} exceeds the limitation {} '
                     'configured in reactor config.'
                     .format(image_size, dest_tag, image_size_limit)
                 )
+                write_task_error(task_result, repr(exc))
```

Comment:
Instead of handling this individually in 10 different places in the build task, could handle this in a single place in Task.run? https://github.com/containerbuildsystem/atomic-reactor/blob/e96d1b89b268731a294a9b7d6495de49f41178aa/atomic_reactor/tasks/common.py#L123-L135

Add an `except` block, check if the param for writing error message was set, write error.

Reply:
mmm I guess it should work

Reply:
done